demisto_sdk/commands/postman_codegen/postman_codegen.py

```diff
@@ -360,11 +360,14 @@ def convert_request_to_command(item: dict):
             try:
                 body_obj = json.loads(request_body.get('raw'))
                 body_format = create_body_format(body_obj)
+                flattened_json = flatten_json(body_obj)
+                shared_args_path = find_shared_args_path(flattened_json)
 
-                for key, value in flatten_json(body_obj).items():
+                for key, value in flattened_json.items():
                     path_split = key.split('.')
                     json_path = path_split[:-1]
-                    arg_name = path_split[-1]
+                    path_length = shared_args_path[path_split[-1].lower()] + 1
+                    arg_name = '_'.join(path_split[-path_length:])
```

Comment:
Please change these two lines to make it clearer. Either change the path_length argument name to something like:  `min_unique_path_length` or separate `path_split[-path_length:]` to another argument